### DISCUSSION TITLE
Examples for Neovis.js 2.0.0

### DISCUSSION THREAD

**Discussion by loneamarok72** [neo4j-contrib/neovis.js#271] [General]
Hi,

Can you update the examples with 2.0.0 code please? Would help a lot!

Thanks,
Sriram

**Comment by thebestnom** [comment]
Im pretty sure I updated all of them, which one specifically you mean?

**Reply by loneamarok72** [reply]
Hi! Thanks for the reply! Since the code in the MIgrating guide at https://github.com/neo4j-contrib/neovis.js/blob/2.0.0/MIGRATING.md is different from the examples, I thought it is yet to be updated. (I was just trying the simple_example.html and was getting a div with no errors with the existing code)
